docs/understanding-your-pipeline/destinations/index.md

```diff
@@ -15,7 +15,7 @@ Here you will find an overview of what’s available.
 
 ## Data warehouses and lakes
 
-Snowplow is primarily built for data warehouse and lake destinations and supports Redshift, BigQuery, Snowflake and Databricks, as well as S3 and GCS, via the [various loaders](/docs/storing-querying/storage-options/index.md).
+Snowplow is primarily built for data warehouse and lake destinations and supports Redshift, BigQuery, Snowflake, Databricks and Synapse Analytics, as well as S3, GCS and ADLS, via the [various loaders](/docs/storing-querying/storage-options/index.md).
```

Comment:
Are we missing Azure Fabric from this list or is that inferred from ADLS?

Reply:
Added OneLake — with Fabric things get a bit muddy because it’s not really a warehouse. But since OneLake is a part of Fabric, it will imply Fabric.